src/pages/index.js

```diff
@@ -118,19 +194,40 @@ function getCardElement(data) {
 
   const cardNameElem = cardElement.querySelector(".card__title");
   const cardDeleteBtn = cardElement.querySelector(".card__delete-btn");
-  cardNameElem.textContent = data.name;
-
   const cardImage = cardElement.querySelector(".card__image");
   const cardLikeBtn = cardElement.querySelector(".card__like-btn");
+  if (data.isLiked) {
+    cardLikeBtn.classList.add("card__like-btn_liked");
+  }
+  cardNameElem.textContent = data.name;
   cardImage.src = data.link;
   cardImage.alt = data.name;
 
-  cardLikeBtn.addEventListener("click", () => {
-    cardLikeBtn.classList.toggle("card__like-btn_liked");
+  function handleLikeBtn(evt, id) {
+    const isLiked = evt.target.classList.contains("card__like-btn_liked");
+
+    console.log(isLiked);
+
+    api
+      .handleLikeCard(id, isLiked)
+      .then(() => {
+        evt.target.classList.toggle("card__like-btn_liked");
+      })
+      .catch((err) => {
+        console.error(err);
+      });
+  }
+
+  cardLikeBtn.addEventListener("click", (evt) => {
+    handleLikeBtn(evt, data._id, data.isLiked);
   });
 
-  cardDeleteBtn.addEventListener("click", () => {
-    cardElement.remove();
+  cardDeleteBtn.addEventListener("click", (evt) => {
+    handleDeleteCard(cardElement, data._id);
+  });
+
+  profileAvatarBtn.addEventListener("click", () => {
+    openModal(profileAvatarModal);
   });
```

Comment:
The profile opening handler should **not be** inside `getCardElement`


Please, remove it from `getCardElement`

The profile opening should be handled only **once** in the file body, otherwise you add listeners again and again to the same elements. This may cause **`memory leak`**